automation/taskcluster/lib/tasks.py

```diff
@@ -615,9 +628,9 @@ def _craft_raptor_task(
                     "./test-linux.sh",
                     "--cfg=mozharness/configs/raptor/android_hw_config.py",
                     "--test={}".format(test_name),
-                    "--app=fenix",
-                    "--binary=org.mozilla.fenix.performancetest",
-                    "--activity=org.mozilla.fenix.browser.BrowserPerformanceTestActivity",
+                    "--app={}".format(_PROJECT_NAME),
```

Comment:
I don't think the `format(...)` is needed here

Reply:
I'm not sure to follow. Do you want `"--app=" + _PROJECT_NAME`, instead?

Reply:
I'm confused, too. I read this as `"{}".format(_PROJECT_NAME)`, sorry!